CK2Plus_expanded/history/provinces/1660 - Chlisselbourg.txt

```diff
@@ -0,0 +1,28 @@
+# 1660 - Chlisselbourg
+
+# County Title
+title = c_chlisselbourg
+
+# Settlements
+max_settlements = 1
+
+b_chlisselbourg = tribal	#1323
+
+#b_mokrishvitsa

```

Comment:
Is there a functional difference between `#some_text` and `# some_text` (note the space)?

Reply:
There is not. In paradox script, anything after a `#` is treated as part of the comment, similar to `//` in other languages.

Reply:
OK, I thought they were both just comments, but the formatting was different so I was unsure. Thanks!